js/app.js

```diff
@@ -0,0 +1,275 @@
+      suit: 'diamonds'
+    },
+    {
+      priority: 3,
+      id: '4-diamonds',
+      suit: 'diamonds'
+    },
+    {
+      priority: 4,
+      id: '5-diamonds',
+      suit: 'diamonds'
+    },
+    {
+      priority: 5,
+      id: '6-diamonds',
+      suit: 'diamonds'
+    },
+    {
+      priority: 6,
+      id: '7-diamonds',
+      suit: 'diamonds'
+    },
+    {
+      priority: 7,
+      id: '8-diamonds',
+      suit: 'diamonds'
+    },
+    {
+      priority: 8,
+      id: '9-diamonds',
+      suit: 'diamonds'
+    },
+    {
+      priority: 9,
+      id: '10-diamonds',
+      suit: 'diamonds'
+    },
+    {
+      priority: 10,
+      id: 'jack-diamonds',
+      suit: 'diamonds'
+    },
+    {
+      priority: 11,
+      id: 'queen-diamonds',
+      suit: 'diamonds'
+    },
+    {
+      priority: 12,
+      id: 'king-diamonds',
+      suit: 'diamonds'
+    },
+    {
+      priority: 0,
+      id: 'ace-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 1,
+      id: '2-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 2,
+      id: '3-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 3,
+      id: '4-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 4,
+      id: '5-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 5,
+      id: '6-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 6,
+      id: '7-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 7,
+      id: '8-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 8,
+      id: '9-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 9,
+      id: '10-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 10,
+      id: 'jack-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 11,
+      id: 'queen-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 12,
+      id: 'king-clubs',
+      suit: 'clubs'
+    },
+    {
+      priority: 0,
+      id: 'ace-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 1,
+      id: '2-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 2,
+      id: '3-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 3,
+      id: '4-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 4,
+      id: '5-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 5,
+      id: '6-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 6,
+      id: '7-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 7,
+      id: '8-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 8,
+      id: '9-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 9,
+      id: '10-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 10,
+      id: 'jack-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 11,
+      id: 'queen-spades',
+      suit: 'spades'
+    },
+    {
+      priority: 12,
+      id: 'king-spades',
+      suit: 'spades'
+    }
+  ];
+
```

Comment:
Other possible solution is to create 2 objects - one with suites(4 child objects) and one with priority (13 child objects). It will take less code and you could do same thing with 2 loops.